### PR TITLE
added new library primes.1.0

### PR DESCRIPTION
**Pull request by KitFreddura**
Wrote a small library for dealing with prime numbers, thanks!

### CODE REVIEW

File: packages/primes/primes.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+dev-repo: "https://github.com/KitFreddura/OCaml-Primes.git"
+homepage: "https://github.com/KitFreddura/OCaml-Primes"
+bug-reports: "None yet"
+build: [
+  ["mkdir" "%{prefix}%/lib/primes"]
+]
+ocaml-version: [>= "4.01"]
+install: [
+["ocamlfind" "ocamlc" "-c" "-linkpkg" "-package" "zarith" "primes.mli"]
+["ocamlfind" "ocamlc" "-thread" "-o" "primes.cma" "-linkpkg" "-package" "zarith,core,gen" "primes.ml"]
```
Comment (yallop) [inline packages/primes/primes.1.0/opam]: These two lines build a bytecode library from `primes.ml`.  You can also build a native code library using 

   [`ocamlfind ocamlopt ...`](http://projects.camlcity.org/projects/dl/findlib-1.2.1/doc/guide-html/c188.html#AEN190).

File: packages/primes/primes.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+["cp" "primes.cmo" "%{prefix}%/lib/primes/"]
+["cp" "primes.cmi" "%{prefix}%/lib/primes/"]
+["cp" "primes.cma" "%{prefix}%/lib/primes/"]
+["cp" "primes.mli" "%{prefix}%/lib/primes/"]
+["cp" "META" "%{prefix}%/lib/primes/"]
+]
```
Comment (yallop) [inline packages/primes/primes.1.0/opam]: Have you considered using [`ocamlfind`'s `install` command](http://projects.camlcity.org/projects/dl/findlib-1.2.1/doc/guide-html/c348.html#AEN350)?

It should be possible to replace these five lines with a single call to `ocamlfind install ...`

File: packages/primes/primes.1.0/opam [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+remove: ["ocamlfind" "remove" "primes"]
+depends: [
+  "ocamlfind" {build}
+  "zarith" {build}
+  "core" {build}
+  "gen" {build}
+]
```
Comment (yallop) [inline packages/primes/primes.1.0/opam]: Are these really all build-only dependencies?  If your library relies on any of them at run time then you should remove `{build}`.  It appears that only `ocamlfind` is a build-only dependency.

File: packages/primes/primes.1.0/opam [hunk 4]
```diff
@@ -0,0 +1,30 @@
+
+opam-version: "1.2"
+name: "primes"
+version: "1.0"
+maintainer: "Kit Freddura <kitfreddura@gmail.com>"
+authors: "Kit Freddura <kitfreddura@gmail.com>"
+license: "MIT License"
+dev-repo: "https://github.com/KitFreddura/OCaml-Primes.git"
+homepage: "https://github.com/KitFreddura/OCaml-Primes"
+bug-reports: "None yet"
```
Comment (yallop) [inline packages/primes/primes.1.0/opam]: `bug-reports` should be a URL where bugs can be reported.

**Comment by KitFreddura** [comment]
I appreciate all of this feedback, this is my first time trying to contribute to a repo like opam so this is all very helpful, thanks!

**Comment by avsm** [comment]
Almost there -- you've inculded the older version of the library as well -- do you want to remove it in favour of 1.0.1 only?

File: packages/primes/primes.1.0.1/url [hunk 5]
```diff
@@ -0,0 +1,2 @@
+http: "https://github.com/KitFreddura/OCaml-Primes/archive/1.3.3.tar.gz"
```
Comment (yallop) [inline packages/primes/primes.1.0.1/url]: The directory says `1.0.1`, but the filename is `1.3.3.tar.gz`.  Could you please rename the directory to match the release number?

**Comment by yallop** [comment]
@KitFreddura: Could you please remove the `primes.1.0.1` and `primes.1.0` directories?  Besides that I think this is ready to merge.

**Comment by KitFreddura** [comment]
@yallop all set!

**Comment by yallop** [comment]
Thank you!

**Pull request merged**